l1-contracts/test/fixtures/empty_block_1.json

```diff
@@ -20,12 +20,12 @@
       },
       "globalVariables": {
         "blockNumber": 1,
-        "slotNumber": "0x0000000000000000000000000000000000000000000000000000000000000033",
+        "slotNumber": "0x0000000000000000000000000000000000000000000000000000000000000034",
```

Comment:
Is it essential that these are changed for this pr?

Reply:
Ah, was likely because I was trying out some of the 36 second cases as well. Can revert it.